### DISCUSSION TITLE
Connecting to RXA3030

### DISCUSSION THREAD

**Discussion by jobc82** [mvdwetering/yamaha_ynca#175] [Q&A]
Hello,

I can´t connect to my RXA 3030. I filled in the IP and let the port on default. Maybe the port is wrong?

Thanks in advance Johannes

**Comment by mvdwetering** [comment]
It is unlikely that the port is wrong. As far as I can tell you can't change it through the receiver menus, but only by connecting through the YNCA protocol to begin with (which is an interesting design decision). 

Just to make sure, check that you are on the 7.3.0 version of the integration as it fixed an issue with Python 3.11 / HA 2023.6.0 causing it to be unable to connect at all.

If that all seems ok and it still does not work you can manually enable logging for this integration.
In the `configuration.yaml` add the following section (or extend the existing section if you already had one) and try again. Then attach the `home-assistant.log` file here and I can have a look.

```
logger:
  default: info
  logs:
    ynca: debug
    custom_components.yamaha_ynca: debug
```

After collecting logging you can remove the logging section again.

**Comment by mvdwetering** [comment]
One more thing to check is that you make sure that "Network Standby" is "On", otherwise the receiver does not accept network connections when it is Off. You can find it in the receiver settings Setup > Network

**Comment by jobc82** [comment]
Thanks 










I made the update, now it works!
So I can confirm that the RX-A3030 is working.

Thanks!

**Comment by mvdwetering** [comment]
Cool, nice to hear you got it working. 
Adding the RX-A3030 to the list 👍
